src/utility.cpp

```diff
@@ -515,106 +558,128 @@ BOOL PatchImport (HMODULE importmodule, moduleentry_t *patchModule)
 	LPSTR pszBuffer = cwBuffer;
 	DWORD dwMaxChars = _countof(cwBuffer);
 	DWORD dwLength = ::GetModuleFileNameA(importmodule, pszBuffer, dwMaxChars);
-#endif
+#endif
+
+    // have a stack local array of the addresses, don;t want to use malloc for this

```

Comment:
>; [](start = 53, length = 1)

nit: apostrophe

Reply:
Fixed.

---
In reply to: [591985218](https://github.com/Azure/vld/pull/8#discussion_r591985218) [](ancestors = 591985218)